kubernetes-model-generator/kubernetes-model-admissionregistration/pom.xml

```diff
@@ -32,6 +32,83 @@
 
   <properties>
     <clone-kube>true</clone-kube>
+    <osgi.include.resources>
+      ${osgi.include.resources.default},
+      /core.properties=target/classes/core.properties,
```

Comment:
This doesn't look right. It should be the same as the values provided in `<Include-Resource>` plugin configuration which you deleted below.